.travis.yml

```diff
@@ -1,36 +1,32 @@
+dist: xenial
 language: python
 python:
   - 3.4
   - 3.5
   - 3.6
-  - nightly
+  - 3.7
 
 matrix:
   # allow failing builds on specified python versions
   allow_failures:
     - python: 3.4
+    - python: 3.5
+    - python: 3.6
 
 # environment variables
 env:
  - TESTING=True
 
 install:
   - sudo apt-get install libvirt-dev
-  - pip install codecov
-  - pip install nose-cov
-  - pip install pycodestyle
-  - pip install pydocstyle
-  - pip install dodgy
-  - pip install isort
-  - pip install mypy
+  - pip install -r test-requirements.txt
   - pip install -r requirements.txt
-  - pip install Flask-Testing
-  - pip install blinker
 script:
   - dodgy
-  - mypy mod_*
-  - pydocstyle --config=./.pydocstylerc
   - isort -rc --check-only
+  - pydocstyle --config=./.pydocstylerc
```

Comment:
Why call this twice with minimal changes in how it's called?

Reply:
One is py**doc**style and the other is py**code**style